slides/08-Platform-Hub/1020-LetsGetOnboarded.md

```diff
@@ -0,0 +1,11 @@
+## Let's get onboarded!
+
+Please follow these step to onboard yourself to the Platform Hub:
+
+ 1. Go to https://access-acp.digital.homeoffice.gov.uk to download and run `ACP NotProd` VPN profile.
```

Comment:
can we make these links e.g. [access-acp.digital.homeoffice.gov.uk](https://access-acp.digital.homeoffice.gov.uk) ?